.freeipa-pr-ci.yaml

```diff
@@ -0,0 +1,24 @@
+jobs:
+  fedora-25/build:
```

Comment:
Why won't we run it on F26 at least? We should probably actually test on rawhide but that, we know, is currently broken (although there is a copr repo which should help us work around it).